themes/bootstrap3/templates/Helpers/email-form-fields.phtml

```diff
@@ -38,3 +38,10 @@
 <div class="form-group">
   <input type="submit" class="btn btn-primary" name="submit" value="<?=$this->transEscAttr('Send')?>"/>
 </div>
+<?php $emailFormScript = <<<'JS'
+    $('#email_to, #email_from, #email_subject').on('invalid', function($e) {
+        $('#modal .loading-spinner').remove();
```

Comment:
Thanks for fixing the class, but do we need this code at all? It was removed by PR #2274, so I'm guessing it got added back by accident.